app/entities/user.go

```diff
@@ -87,3 +94,11 @@ func checkPass(password string) error {
 
 	return nil
 }
+
+func (u UserID) ValidateUUID() error {
+	if valid := IDRegex.MatchString(u.ID); !valid {
+		return fmt.Errorf("ID does not match with regex: `%s`", IDMask)
```

Comment:
If you write user id instead of ID - it will have 2 benefits:
1. linters will be happy (cos error msg should start with lowercase letter)
2. user id has more context than just id, so it will be a bit more easier to read

Reply:
Fixed